picky/src/http/http_signature.rs

```diff
@@ -272,6 +298,14 @@ impl FromStr for HttpSignature {
             None
         };
 
+        let signature =
+            keys.remove(HTTP_SIGNATURE_SIGNATURE)
+                .ok_or_else(|| HttpSignatureError::MissingRequiredParameter {
+                    parameter: HTTP_SIGNATURE_SIGNATURE,
+                })?;
+
+        let legacy = !signature.ends_with('=');
```

Comment:
Could we have case where we don't have to add padding in base64 "normal" mode ? I think it is possible, no ?

Reply:
Should never happen with SHA256, but might indeed happen with something else. Let's be safe, I'll check for '/' and '+' characters instead.